internal/infrastructure/runner/runner.go

```diff
@@ -79,6 +79,9 @@ func (r *Runner) subscribeToRateLimitInfraIR(ctx context.Context) {
 		func(update message.Update[string, *ir.RateLimitInfra]) {
 			val := update.Value
 
+			// loading EnvoyGateway Configurations.
+			val.Config = r.EnvoyGateway
```

Comment:
why is this needed, `EnvoyGateway` is a static config, set during startup, and it doesnt change, so it shouldnt be part the pub-sub messages